src/libraries/Common/src/Interop/Browser/Interop.JavaScript.Array.cs

```diff
@@ -0,0 +1,44 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
```

Comment:
Nit: The usual style is to have a empty new line after the `using`s block

Reply:
Addressed in recent commit